main.go

```diff
@@ -1,49 +1,11 @@
 package main
 
 import (
-	"flag"
-	"fmt"
-	"os"
-	"sort"
-
-	"golang.org/x/tools/go/packages"
-
 	"github.com/polyfloyd/go-errorlint/errorlint"
+
+	"golang.org/x/tools/go/analysis/singlechecker"
```

Comment:
Could you please ensure 3rd party imports are after Go stdlib imports but before project imports?